src/main/java/com/github/dockerjava/api/model/ResponseItem.java

```diff
@@ -16,57 +16,83 @@
 
     private static final long serialVersionUID = -5187169652557467828L;
 
+    @JsonProperty("stream")
+    private String stream;
+
     @JsonProperty("status")
     private String status;
 
+    @JsonProperty("progressDetail")
+    private ProgressDetail progressDetail;
+
     @JsonProperty("progress")
     private String progress;
 
-    @JsonProperty("progressDetail")
-    private ProgressDetail progressDetail;
+    @JsonProperty("id")
+    private String id;
 
-    @JsonProperty("error")
-    private String error;
+    @JsonProperty("from")
+    private String from;
+
+    @JsonProperty("time")
+    private long time;
 
     @JsonProperty("errorDetail")
     private ErrorDetail errorDetail;
 
-    @JsonProperty("id")
-    private String id;
+    @JsonProperty("error")
+    private String error;
 
-    public String getStatus() {
-        return status;
+    public String getStream() {
+        return stream;
     }
 
-    public String getProgress() {
-        return progress;
+    public String getStatus() {
+        return status;
     }
 
     public ProgressDetail getProgressDetail() {
         return progressDetail;
     }
 
+    @Deprecated
+    public String getProgress() {
```

Comment:
Talk to the docker people. Anyway, it's kind of obvious what the replacement is, no? ProgressDetail.